src/api.js

```diff
@@ -25,12 +25,8 @@ export const api = {
           { id: 3, title: 'Most Popular', active: false },
           { id: 4, title: 'Free Books', active: false },
         ];
-        const params = {
-          search: search || null,
-          activeCategory: null,
-          activeFilter: getActiveFilter({ filters }),
-        };
-        fetch(`http://localhost:${PORT}/api/books?` + concatUrlParams(params))
```

Comment:
@W01fw00d can you update `concatUrlParams` function to function that concat only not null params?
And revert this changes please.

Reply:
Ok, activeCategory is always null, so we can remove that one right? But we want to always use the activeFilter as a param, and "search" in case it isn't null.